config/coil_player.rst

```diff
@@ -85,6 +85,8 @@ If not set it will use ``default_pulse_power`` of the :doc:`/config/coils`.
 
 pulse_ms:
 ~~~~~~~~~
+Single value, type: float(0,1).
```

Comment:
This is an integer. You cannot add fractions. This row is also auto-generated in the next version of the docs (first line of an attribute) so better don't change it manually